site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -50,14 +50,104 @@ public function editGradeableRequest($gradeable_id, $nav_tab = 0) {
         'electronic_bulk' => 'TA/Instructor will (bulk) upload scanned .pdf for online manual grading'
     ];
 
+    /**
+     * @Route("/api/{_semester}/{_course}/{gradeable_id}/download", methods={"GET"})
+     * @Route("/courses/{_semester}/{_course}/{gradeable_id}/download", methods={"GET"})
+     */
+    public function downloadJson(string $gradeable_id) {
```

Comment:
One side effect of this decision is that there is now an API route which does not return data in the standard success/failure format.  I feel like that kind of inconsistency could be problematic in the future.  Do you think it would be a good idea to make two routes: one regular download route which keeps the logic as-is, and an API route which packages the data up in a success response?